app/src/main/java/com/nervousfish/nervousfish/ConstantKeywords.java

```diff
@@ -0,0 +1,15 @@
+package com.nervousfish.nervousfish;
+
+/**
+ * Contains the keywords used in the application.
+ */
+public final class ConstantKeywords {
+    public static final String SERVICE_LOCATOR = "a";
```

Comment:
In EntryActivity we pass the service locator to the mainactivity, that must be identified by a certain string. By using a constant String we won't have to worry about typing errors